fahari/common/models/common_models.py

```diff
@@ -280,7 +280,7 @@ class SystemPatters(models.TextChoices):
 
     name = models.CharField(max_length=128, null=False, blank=False, unique=True)
     pattern = models.CharField(
-        max_length=10, choices=SystemPatters.choices, default=SystemPatters.NONE.value
+        max_length=100, choices=SystemPatters.choices, default=SystemPatters.NONE.value
```

Comment:
For this one too, 10 characters were enough. And it also more efficient this way.